docs/source/releases/v2.0.rst

```diff
@@ -237,83 +247,95 @@ Backwards incompatible changes in Oscar 2.0
-    ``oscar.core.application.OscarDashboardConfig`` subclass in the ``apps.py``
-    module.
-    - These include the following overridden attributes:
-      - Rename ``name`` to ``namespace``
-      - ``login_url``
-      - ``hidable_feature_name``
-      - ``permissions_map``
-      - ``default_permissions``
-    - These include the following overridden methods:
-      - :meth:``get_urls``
-      - :meth:``post_process_urls``
-      - :meth:``get_permissions``
-      - :meth:``get_url_decorator``
-      - :meth:``urls``
-  - For URLconfs that include Oscar app URLs (e.g. in the ``dashboard`` app):
-    - Obtain the app's Django/Oscar app config instance from the Django app
-      registry, instead of from the app's previous config module's (``app.py``)
-      ``application`` variable.
-    - If the to-be-included URLs have an instance namespace, use the form
-      ``app_config.urls``, which passes in the list of URL patterns, the app
-      namespace, and the instance namespace.
-    - If the to-be-included URLs do not have an instance namespace, use the
-      form ``include(app_config.urls[0])``, which only passes in the list of
-      URL patterns.
-
-- For the main app's URLconf:
-  - Obtain the ``oscar`` app's Django/Oscar app config instance from the Django
-    app registry, instead of from the previous config module's (``config.py``)
-    ``application`` variable.
-  - Include the ``oscar`` app's URLs using the form
-    ``include(app_config.urls[0])``, to only pass in the list of URL patterns,
-    and not an app or instance namespace.
+In release 2.0 the way apps are configured has been substantially refactored to
+merge Oscar's ``Application`` class with Django's ``AppConfig`` class. For
+each app that you have forked, you will need to:
+
+- Rename the ``config.py`` module to ``apps.py``.
+
+- Change the ``default_app_config`` variable in ``__.init__.py`` to point
+  to ``apps`` instead of ``config``.
+
+- Update your ``AppConfig`` subclass in the ``apps.py`` module to either
+  inherit from the parent app's ``AppConfig`` or use either
+  ``oscar.core.application.OscarConfig`` or
+  ``oscar.core.application.OscarDashboardConfig``.
+
+- Move any changes you've made to the Oscar ``Application`` subclass in the ``app.py``
+  module, to the ``AppConfig`` subclass in the ``apps.py`` module.
+
+- Rename the ``name`` attribute to ``namespace``.
+
+- In your project URLconf and in `get_urls` methods replace the ``application``
+  import by finding the app in the Django app registry.
+
+.. code-block:: python
+
+    from django.apps import apps
+    application = apps.get_app_config('your_app_name')
+
+- If the urls you're including don't define an instance namespace then use
+  ``include(application.urls[0])``, which only passes in the list of URL
+  patterns.
 
 Dependency changes
 ------------------
 
-- Upgraded TinyMCE to version 4.8.3.
+- Dropped ``mock`` as a dependency in favour of ``unittests.mock``.
```

Comment:
Several of these are node JS dependencies which are only used for development - I suggest we don't list those here. Specifically:
- extend
- gulp-*
- require-dir